lib/model/parameter/Barcodes.dart

```diff
@@ -0,0 +1,16 @@
+import 'package:openfoodfacts/interface/Parameter.dart';
+
+/// "Barcodes" search API parameter.
+class Barcodes extends Parameter {
```

Comment:
Thats a bit generic, what about BarcodeParameter

Reply:
@M123-dev Good point. "Filter" would even be better, but the rest is rather named "Parameter".
Renamed.